lib/commands/performance.js

```diff
@@ -3,19 +3,31 @@ import { retryInterval } from 'asyncbox';
 import log from '../logger';
 
 
-let commands = {}, helpers = {}, extensions = {};
+let commands = {}, helpers = {}, extensions = {}, memoryParsers = {};
 
 const NETWORK_KEYS = [['bucketStart', 'activeTime', 'rxBytes', 'rxPackets', 'txBytes', 'txPackets', 'operations', 'bucketDuration'], ['st', 'activeTime', 'rb', 'rp', 'tb', 'tp', 'op', 'bucketDuration']];
 const CPU_KEYS = ['user', 'kernel'];
 const BATTERY_KEYS = ['power'];
-const MEMORY_KEYS = ['totalPrivateDirty', 'nativePrivateDirty', 'dalvikPrivateDirty', 'eglPrivateDirty', 'glPrivateDirty', 'totalPss', 'nativePss', 'dalvikPss', 'eglPss', 'glPss', 'nativeHeapAllocatedSize', 'nativeHeapSize'];
-
+const MEMORY_KEYS = ['totalPrivateDirty', 'nativePrivateDirty', 'dalvikPrivateDirty', 'eglPrivateDirty', 'glPrivateDirty',
+  'totalPss', 'nativePss', 'dalvikPss', 'eglPss', 'glPss',
+  'nativeHeapAllocatedSize', 'nativeHeapSize',
+  'nativeRss', 'dalvikRss', 'totalRss'];
 const SUPPORTED_PERFORMANCE_DATA_TYPES = {
   cpuinfo: 'the amount of cpu by user and kernel process - cpu information for applications on real devices and simulators',
   memoryinfo: 'the amount of memory used by the process - memory information for applications on real devices and simulators',
   batteryinfo: 'the remaining battery power - battery power information for applications on real devices and simulators',
   networkinfo: 'the network statistics - network rx/tx information for applications on real devices and simulators'
 };
+const MEMORY_CONSTANT = {
```

Comment:
I usually use Object.freeze on the value to make sure it won't be eventually modified